cmd/cli/uninstall_mesh.go

```diff
@@ -179,6 +179,22 @@ func (d *uninstallMeshCmd) run() error {
 			}
```

Comment:
to fix the lint, let's try to reduce some of the nesting. How about at the end of this err != nil check, add a continue, and then we can remove the `if err == nil` check.

Reply:
Hi @steeling thanks for the review I have removed the check